### ISSUE TITLE
Some buttons do not show their cursor state

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#6443]
**Describe the bug**
When using Quality-time without the use of a mouse, some buttons do not show that they are selected. This can cause confusion for the end user since he/she cannot determine their position.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to a Quality-time report
2. Disable or put away your mouse
3. Try to navigate with the keyboard using tab
4. As soon as you have selected the triangle at the start of a metric row, hit enter
5. continue to tab through the metric configuration elements
6. after 'metric near target' the cursor will enter the blue 'move metric' buttons, I cannot tell if these buttons are selected and they also do not show their pop-up.
7. when you continue to tab to the 'delete metric' button, the user can only tell (guess) by the pop-up that this button is now selected

**Expected behaviour**
It would be useful if the selected buttons show their pop-up and change their colour scheme so that physically challenged people know what is happening in the navigation.

**Screenshots**
![image](https://github.com/ICTU/quality-time/assets/95095453/873b0052-c086-4e9b-9627-56f994918b4d)

**Suggestion**
![image](https://github.com/ICTU/quality-time/assets/95095453/a44476ac-44ec-44ed-acad-856ccee0fb91)

**Comment by fniessink** [comment]
Blocked by #9550.
